core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -48,7 +48,7 @@ oppia.factory('EditableQuestionBackendApiService', [
         skillId, questionDict, successCallback, errorCallback) {
       var questionCreationUrl = UrlInterpolationService.interpolateUrl(
         QUESTION_CREATION_URL, {
-          skill_id: skillId
+          skill_ids: skillId
```

Comment:
This seems wrong, should it be [skillId]?

Reply:
No it can be a string or a comma separated list of strings

Reply:
Wait sorry I think I didn't explain clearly
skillId will be a string, that consists of a comma separated list of strings. As of now since we pass only one skill ID in the frontend, we have that string is just one skill ID, but after making frontend changes we will have a string that is a comma separated list of strings

Reply:
Wait sorry I think I didn't explain clearly
skillId will be a string, that consists of a comma separated list of strings. As of now since we pass only one skill ID in the frontend, we have that string is just one skill ID, but after making frontend changes we will have a string that is a comma separated list of strings